File: week1/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_pyramid(int n);

int main(void)
{
    // Get height from user in the range <1,8>
    static int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    // Print a pyramid with the provided height
    for (int i=0; i < height; i++)
    {
        for (int j=0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        print_pyramid(i);
        printf(" ");

        print_pyramid(i);
        printf("\n");
    }
}

void print_pyramid(int n)
{
    for (int j=0; j < n + 1; j++)
    {
        printf("#");
    }
}

File: week6/pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        if height <= 8 and height >= 1:
            for i in range(height):
                print(" " * (height - i - 1), end="")
                print_hashes(i)
                print("  ", end='')

                print_hashes(i)
                print()

            break


def print_hashes(n):
    print("#" * (n + 1), end='')


main()
